rest/src/main/java/fi/otavanopisto/pyramus/rest/StudentRESTService.java

```diff
@@ -1850,6 +1913,179 @@ public Response deleteStudentContactLogEntry(@PathParam("STUDENTID") Long studen
 
     return Response.noContent().build();
   }
+  
+  @Path("/students/{STUDENTID:[0-9]*}/contactLogEntry/{ID:[0-9]*}/entryComments")
+  @GET
+  @RESTPermit(StudentContactLogEntryPermissions.FIND_STUDENTCONTACTLOGENTRY)
+  public Response listStudentContactLogEntryCommentsById(@PathParam("STUDENTID") Long studentId, @PathParam("ID") Long id) {
```

Comment:
This endpoint wouldn't be needed at all if contact log entry comments are returned with contact log entries themselves.